client/src/mods/Panels.lua

```diff
@@ -441,21 +445,24 @@ local function getGarbageBounceProps(panelSet, panel)
   end
 end
 
-local function getDangerBounceProps(panelSet, panel, dangerTimer)
+local function getDangerBounceProps(panelSet, panel, dangerTimer, stopTime)
```

Comment:
I've been thinking a bit in terms of API design.
Why does it need the actual stop time instead of just a bool of whether there is stop or not?
Then I figured maybe the threshold for switching could be made configurable too, given that the practical consequences of having 1 or 0 frames of stop time are...relatively similar.
Thoughts?